### PR TITLE
fixed formatting of court address and added question about which court

### LINKED ISSUE #440: Fix formatting of court name/address/hours on `id: download eviction_defender`
`  ${ county_dictionary[trial_court.county]['court_address_and_operating_hours'] }`

### LINKED ISSUE #542: Counties that have two courts default to one court

### PR DESCRIPTION
**Pull request by tobyfey**
<Type out your reasons for this PR>

I changed the formatting of county_dictionary to allow more information about courts and to provide the right information when there are more than two courts in one county.  I added to the court name to ask for a location name if it is appropriate, and I added code to update trial_court to the appropriate court to use trial court and specific attributes on the download page.

<Add links to any solved or related issues here>
close #440 
close #542 
### In this PR, I have:

<Check these boxes below before making the PR>
<To turn the box into a checkbox add an X inside it like this [X]>
<Remove spaces from the checkboxes so it's like this [X] not this [X ]>
<To Preview what your PR will look like, select "Preview" above>

* [x] Manually tested to ensure my PR is working
* [x] Ensured issues that this PR closes will be [automatically closed](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
* [x] Requested review from Mia or Quinten
* [ ] Ensured automated tests are passing
* [ ] Updated automated tests so they are now passing
* [ ] There were no automated tests on this repo so I filled out [this interview](https://apps-dev.suffolklitlab.org/run/test-setup/) and there is now an "it runs" test

### CODE REVIEW

File: docassemble/MOHUDEvictionProject/data/questions/court_information.yml [hunk 1]
```diff
@@ -13,9 +13,46 @@ code: |
...
+    for index in range(len(county_dictionary[trial_court.county]["courts"])):
+      potential_courts.append({index:county_dictionary[trial_court.county]["courts"][index]["location_name"]})
   trial_court.division = "Associate Division"
   trial_court.address.county = trial_court.county
 ---
+code: |
+  if len(county_dictionary[trial_court.county]["courts"]) == 1:
+    trial_court.address.address = county_dictionary[trial_court.county]["courts"][0]["address"]
+    trial_court.building_name = county_dictionary[trial_court.county]["courts"][0]["name"]
+    if "unit" in county_dictionary[trial_court.county]["courts"][0].keys():
```
Comment (nonprofittechy) [inline docassemble/MOHUDEvictionProject/data/questions/court_information.yml:29]:
```suggestion
    if "unit" in county_dictionary[trial_court.county]["courts"][0]:
```

`in` automatically implies searching `keys` fwiw

File: docassemble/MOHUDEvictionProject/data/questions/court_information.yml [hunk 2]
```diff
@@ -13,9 +13,46 @@ code: |
...
+    trial_court.address.state = county_dictionary[trial_court.county]["courts"][0]["state"]
+    trial_court.address.zip = county_dictionary[trial_court.county]["courts"][0]["zip"]
+    if "schedule" in county_dictionary[trial_court.county]["courts"][0].keys():
+      trial_court.schedule = county_dictionary[trial_court.county]["courts"][0]["schedule"]
+    else:
+      trial_court.address.schedule = None
+  else:
+    trial_court.building_name = county_dictionary[trial_court.county]["courts"][int(trial_court.location_number)]["name"]
+    trial_court.address.address = county_dictionary[trial_court.county]["courts"][int(trial_court.location_number)]["address"]
+    if "unit" in county_dictionary[trial_court.county]["courts"][int(trial_court.location_number)].keys():
```
Comment (nonprofittechy) [inline docassemble/MOHUDEvictionProject/data/questions/court_information.yml:43]:
```suggestion
    if "unit" in county_dictionary[trial_court.county]["courts"][int(trial_court.location_number)]:
```

File: docassemble/MOHUDEvictionProject/data/questions/court_information.yml [hunk 3]
```diff
@@ -13,9 +13,46 @@ code: |
...
+      trial_court.address.unit = county_dictionary[trial_court.county]["courts"][int(trial_court.location_number)]["unit"]
+    else:
+      trial_court.address.unit = None
+    trial_court.address.city = county_dictionary[trial_court.county]["courts"][int(trial_court.location_number)]["city"]
+    trial_court.address.state = county_dictionary[trial_court.county]["courts"][int(trial_court.location_number)]["state"]
+    trial_court.address.zip = county_dictionary[trial_court.county]["courts"][int(trial_court.location_number)]["zip"]
+    if "schedule" in county_dictionary[trial_court.county]["courts"][int(trial_court.location_number)].keys():
```
Comment (nonprofittechy) [inline docassemble/MOHUDEvictionProject/data/questions/court_information.yml:50]:
```suggestion
    if "schedule" in county_dictionary[trial_court.county]["courts"][int(trial_court.location_number)]:
```

**Review by nonprofittechy — APPROVED** [review]
LGTM if you've feel confident you've sufficiently manually tested the different combinations of filled in and unfilled in court fields in this change--and I'm assuming you automatically generated this rather than manually so I don't need to do a careful check for typos?

**Comment by tobyfey** [comment]
Yes, this was generated from a database and all the fields that aren't conditional are always used.

**Pull request merged**